sys/stdio_nimble/README.md

```diff
@@ -86,7 +86,8 @@ SERVICE e6d54866-0292-4779-b8f8-c52bbec91e71 (Handle: 10): Unknown
 Completed deep scan of 6BE8174C-A0F8-4479-AFA6-9828372CAFE9
 ```
 
-- Create a virtual port and mount it on /tmp/dev_riot_ble
+- Create a virtual port and mount it on /tmp/dev_riot_ble\
+**NOTE:** with `ble-serial` version `>=3.0.0`, the command line option `--write-with-response` has to be added!
```

Comment:
Is it possible to not have the latest version of this when it's installed via `pip` as described above?

Reply:
Yes, you can use 'pip install --force-reinstall -v "ble-serial==2.1.0"' to install an older version.
But by default `pip` will always install the latest version.

My thought behind the sentence was that some people might already have `ble-serial` installed and the command line option `--write-with-response` will probably not work.

Reply:
I would think that people reading this don't yet have this installed, otherwise they would likely already know what do do.

So I'd document the *current* behavior and maybe add a note that the command differed in previous versions.

Reply:
I changed the note in the fixup and I'll prepare another commit to move the whole file to the header, as you proposed.